internal/v2/application/command.go

```diff
@@ -491,28 +493,28 @@ func createCommandValueFromDeviceResource(dr *contract.DeviceResource, v string)
 
 	origin := time.Now().UnixNano()
 	switch strings.ToLower(dr.Properties.Value.Type) {
-	case strings.ToLower(dtos.ValueTypeString):
+	case strings.ToLower(v2.ValueTypeString):
```

Comment:
These un-related changes cause by latest go-mod-core-contracts.

Reply:
IMO, minor names changes don't justify it when can be handle here. Would agree if required logic changes.